src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        for (int i = 0; i < figures.length; i++) {
+            if (i < 3) {
+                figures[i] = new FigureSupplier().getRandomFigure();
+                figures[i].draw();
+            } else {
+                figures[i] = new FigureSupplier().getDefaultFigure();
+                figures[i].draw();
+            }
+        }
```

Comment:
1) Better to minimize hardcoded values to be able to change code for new required size easier
2) please extract common part from if/else statement
```suggestion
        Figure[] figures = new Figure[FIGURES_NUMBER];
        for (int i = 0; i < figures.length; i++) {
            if (i < FIGURES_NUMBER / 2) {
                figures[i] = new FigureSupplier().getRandomFigure();
            } else {
                figures[i] = new FigureSupplier().getDefaultFigure();
            }
            figures[i].draw();
        }
```